src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
+            <sky-docs-anatomy-item anatomyNumber="2" isOptional="true">Image</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="3" isOptional="true">Description</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="4" isOptional="true">Action</sky-docs-anatomy-item>
+          </sky-docs-anatomy>
+    
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Options">
+        
+        <h3>Error type</h3>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the broken error type in situations where the underlying content doesn’t load or when there is an issue reaching the server, such as 500 errors.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-broken.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the construction error type when a page is under maintenance or is experiencing some sort of known outage.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-construction.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the not found error type when there is an error loading a page due to a 404 error.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-notfound.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the security error type when a user doesn’t have access to view content.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-security.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <h3>Image and action</h3>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+                <p>Include the optional image and action when the error component is the only content on the page. Tiles have limited space and may not be important to users' immediate tasks, so exclude the images and actions and only display error messages.</p>
```

Comment:
I'd start with "Only include..." and delete "optional." ... And at the end of the second sentence, I think "error messages" should be "the title and description" to correspond with the anatomy items. I think the "error message" is the entire thing. Or am I thinking about this incorrectly?

Also, the Invitations image at https://developer.blackbaud.com/skyux/design/guidelines/user-assistance#empty-state-help is at odds with the guidance not to include actions in tiles, and I think I've seen several existing tiles where the empty state help includes calls to action. Do we really mean to go against the tide on this?

Reply:
I would consider this difference an intentional distinction between "error handling" and "empty state". The recommended guidelines for prompting user action (with a button) in cases where records are incomplete or could be enhanced feels like it matches our principles well. By contrast, when a portion of page content fails to load because of a system error, prompting user action with the error component's primary (!!) action button doesn't match our principles well.

Maybe we should spell out that distinction here with an example, and link to the empty state guidelines as a way of clarifying that point.

Reply:
Seems like I was getting my wires crossed here and inferring a pattern conflict when it's really 2 distinct patterns. Sorry about that. If you think this is something that might confuse others too, than I agree we could spell out the distinction, but feel free to let me know if you think it's unlikely to be necessary. If we do want to address this, maybe we could add a third sentence to the end along the lines of "This pattern is different than empty state help, which can include actions." And we could make "empty state help" the link to those guidelines. What do you think?

Also, just to follow up on the suggestion to start with "Only include..." and delete "optional." .... As written, it sounds like the guideline is to ALWAYS include the image and action in error messages on pages. If that's the point we are making, I'd split this into 2 paragraphs after the first sentence to make it clearer that the first sentence is a distinct point about pages and that the rest is a distinct point about tiles and such. But if the point we are trying to make is not that images and actions are always required on pages but rather that they should NOT be used in tiles and such, I'd start with "Only include..." and delete "optional."

Reply:
The updates here look good to me!